iotedgehubdev/edgemanager.py

```diff
@@ -193,6 +188,28 @@ def start_solution(self, deployment_config, verbose):
             cmd_up = ['docker-compose', '-f', EdgeManager.COMPOSE_FILE, 'up', '-d']
         Utils.exe_proc(cmd_up)
 
+    @staticmethod
+    def loginRegistries(module_content):
```

Comment:
General suggestion: the whole project uses the `snake_case` pattern of naming for non-class variables.